src/main/create-operator-json/operators/maestro.js

```diff
@@ -9,5 +9,13 @@ let maestro = new Operator(r6operators.maestro, operatorId.maestro, "Operation P
 
 maestro.addCounterNode(operatorId.iana, counterType.soft, "Maestro's Evil Eyes can destroy Iana's Hologram with its laser, and can detect its lack of heat signature.");
 maestro.addCounterNode(operatorId.zero, counterType.soft, "Maestro's Evil Eye can destroy Zero's cameras.");
+maestro.addCounterNode(operatorId.hibana, counterType.hard, "Maestro's Evil Eye can destroy Hibana's X-KAIROS pellets with its laser.")
+maestro.addCounterNode(operatorId.twitch, counterType.hard, "Maestro's Evil Eye can destroy Twitch's shock drones with its laser.")
+maestro.addCounterNode(operatorId.glaz, counterType.minor, "Maestro's Evil Eye can detect Glaz through smoke, which Glaz relies on.")
```

Comment:
Would we be able to remove the minor counter for Glaz, please? I don't think that one is substantial enough.